cargo-nextest/src/dispatch.rs

```diff
@@ -845,9 +845,24 @@ pub struct TestRunnerOpts {
     fail_fast: bool,
 
     /// Run all tests regardless of failure
-    #[arg(long, conflicts_with = "no-run", overrides_with = "fail-fast")]
+    #[arg(
+        long,
+        name = "no-fail-fast",
+        conflicts_with = "no-run",
+        overrides_with = "fail-fast"
+    )]
     no_fail_fast: bool,
 
+    /// Number of tests that can fail before exiting test run
+    #[arg(
+        long,
+        name = "max-fail",
+        value_name = "N",
+        conflicts_with_all = &["no-run", "no-fail-fast"],
+        overrides_with = "fail-fast"
```

Comment:
Hmm is there a reason this overrides `--fail-fast`? I would expect this to conflict with `--fail-fast`, not override it. (Unlike `--no-fail-fast` which is a clear negation.)

Reply:
Hmm, so `--fail-fast` and `--no-fail-fast` overriding each other is part of a long-standing CLI convention that in case of that kind of binary choice, the last argument wins. I feel like `--max-fail` is slightly different from them, though I do see the argument. Ultimately it's a judgment call -- if we change our mind in the future, going from conflicts to overrides is easy while going from overrides to conflicts is harder.